### ISSUE TITLE
Suggestion: A Portal to enter/exit dimension

### ISSUE THREAD

**Issue by Freebi85** [TelepathicGrunt/Bumblezone#35]
Hey There!
Great mod :) Keep up the great work!
A small suggestion from my side:
Give us the possibility to make a proper portal to enter and exit the dimension. Like the nether. 
From a block of your choice or even better configurable in configs for pack devs like me :P

Cheers 🍺

**Comment by TelepathicGrunt** [comment]
Dunno if you are the same person that commented the same question on curseforge lol. Anyway, When I was making this mod, I thought long and hard about how to enter and leave it. I honestly dislike how everyone's dimension mods were basically clones of nether portals and such. Not really original. So I decided to be unique and more immersive. That's why I opted for using ender pearls on beehives to teleport inside the block and then digging out of the dimension to break out of the hive.

I really do not want to add generic portals as it breaks the immersion and the point of my dimension. It also helps balance it a bit so that players have to use up ender pearls and locate beehives in order to enter a dimension that is basically an endless supply of one of the best food in the game lol (honey bottles). However, other mods can provide ways of entering/exiting any dimensions. There might be one already made. I can ask around a bit to see if anyone knows of a mod
